docs/core/compatibility/includes/obsoletion-policy.md

```diff
@@ -0,0 +1,12 @@
+---
+ms.topic: include
+ms.date: 04/26/2023
+---
+
+.NET rarely removes an API due to the large impact it has of breaking compatibility. Removal is done only when there's an unavoidable and justifiable reason. .NET strives to provide mitigations to such impactful breaking changes in advance, for example, by:
+
+- Marking the API as obsolete or providing build diagnostics.
+- Blogging about it.
+- Posting in [dotnet/announcements](https://github.com/dotnet/announcements).
```

Comment:
Instead of dotnet/announcements, it might be better to point to our compatibility breaking changes page https://learn.microsoft.com/en-us/dotnet/core/compatibility/breaking-changes because that  has breaking changes across all repos including ASPNET Core/EF Core/Winforms/WPF etc. I am not sure who updates dotnet/announcements and when.